lib/cli.js

```diff
@@ -37,7 +43,7 @@ async function start() {
   const end = new Date();
   const timeTaken = end.getTime() - start.getTime();
 
-  const resultsPath = path.join(process.cwd(), settings.runnerResults);
+  console.log(`Reading results from ${resultsPath}`);
```

Comment:
both in upstream and in master we create variable resultsPath and use it further, why you decided to change this logic ?